### ISSUE TITLE
One simple question about weight initialization

### ISSUE THREAD

**Issue by wplf** [danielchyeh/ImageNet-100-Pytorch#2]
I found that the params of batchnorm is set as constant 1, neither random nor normal.
Could you tell me why  or throw me a website explaining this problem?
``` python
    def init_params(self):
        for m in self.modules():
            if isinstance(m, nn.Conv2d):
                init.kaiming_normal_(m.weight, mode='fan_out')
                if m.bias is not None:
                    init.constant_(m.bias, 0)
            elif isinstance(m, nn.BatchNorm2d):
                init.constant_(m.weight, 1)
                init.constant_(m.bias, 0)
            elif isinstance(m, nn.Linear):
                init.normal_(m.weight, std=0.001)
                if m.bias is not None:
                    init.constant_(m.bias, 0)
```

**Comment by QianpengLi577** [comment]
ref
https://github.com/pytorch/vision/blob/main/torchvision/models/resnet.py#L212C47-L212C47

**Comment by wplf** [comment]
I get it, that means the  beta and gamma in the formula are 1 and 0.

**Comment by danielchyeh** [comment]
question addressed
